packages/abstract-provider/src.ts/index.ts

```diff
@@ -28,6 +28,7 @@ export type TransactionRequest = {
     maxFeePerGas?: BigNumberish;
     nodeId?: AccountLike,
     customData?: Record<string, any>;
+    isSimpleTransfer?: boolean;
```

Comment:
Let's move the `isSimpleTransfer` field inside the `customData` one. This way, we don't have to add an entirely new property to the `TransactionRequest` object